### PR TITLE
[fix #18] [mozmeao/infra#1104] update image hashes

### LINKED ISSUE #18: fix image in deployments
fix in:

- frankfurt stage/prod
- oregon-b dev/stage

### PR DESCRIPTION
**Pull request by jpetto**
verified no env vars changed between these releases

### CODE REVIEW

File: frankfurt/basket-prod/clock-deploy.yaml [hunk 1]
```diff
@@ -197,7 +197,7 @@ spec:
           value: 4
         - name: conflicts
           value: 1
-        image: mozmeao/basket:66ada14
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline frankfurt/basket-prod/clock-deploy.yaml:200]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: frankfurt/basket-prod/web-deploy.yaml [hunk 2]
```diff
@@ -197,7 +197,7 @@ spec:
           value: 4
         - name: conflicts
           value: 1
-        image: mozmeao/basket:66ada14
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline frankfurt/basket-prod/web-deploy.yaml:200]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: frankfurt/basket-prod/worker-deploy.yaml [hunk 3]
```diff
@@ -198,7 +198,7 @@ spec:
           value: 4
         - name: conflicts
           value: 1
-        image: mozmeao/basket:66ada14
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline frankfurt/basket-prod/worker-deploy.yaml:201]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: frankfurt/basket-stage/web-deploy.yaml [hunk 4]
```diff
@@ -159,7 +159,7 @@ spec:
               name: basket-stage-secrets
         - name: TASK_LOCKING_ENABLE
           value: "True"
-        image: mozmeao/basket:66ada14
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline frankfurt/basket-stage/web-deploy.yaml:162]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

File: frankfurt/basket-stage/worker-deploy.yaml [hunk 5]
```diff
@@ -159,7 +159,7 @@ spec:
               name: basket-stage-secrets
         - name: TASK_LOCKING_ENABLE
           value: "True"
-        image: mozmeao/basket:66ada14
+        image: mozmeao/basket:4d0668d
```
Comment (jgmize) [inline frankfurt/basket-stage/worker-deploy.yaml:162]:
```suggestion
        image: mozmeao/basket:4d0668d4e5cb304e68b04bcc0d2b278d27b591e6
```

**Review by jgmize — CHANGES_REQUESTED** [review]

**Comment by jpetto** [comment]
Closing in favor of #25

**Pull request closed without merging**